tests/test_paging.py

```diff
@@ -49,12 +51,25 @@ def dburl(request):
 
         data.append(b)
 
+    for x in range(count):
+        author = Author(name='Author {}'.format(x))
+        abooks = []
+        for y in range((2*x) % 10):
+            b = Book(name='Book {}-{}'.format(x, y), a=x+y, b=(y*x) % 2, c=count - x, d=99-y)
+            b.author = author
+            if y % 4 != 0:
+                b.prequel = abooks[(2*y+1) % len(abooks)]
+            abooks.append(b)
+            data.append(b)
+
     with temporary_database(request.param) as dburl:
         with S(dburl) as s:
             Base.metadata.create_all(s.connection())
             s.add_all(data)
         yield dburl
 
+dburl = pytest.fixture(params=['postgresql', 'mysql'])(_dburl)
+pg_only_dburl = pytest.fixture(params=['postgresql'])(_dburl)
```

Comment:
There's probably a cleaner/more idiomatic way of doing this - I'm no pytest guru.

Reply:
lgtm